src/file.cpp

```diff
@@ -246,3 +246,12 @@ Artboard* File::artboard() const
 	}
 	return m_Artboards[0];
 }
+
+Artboard* File::artboard(size_t index) const
+{
+	if (index < 0 || index >= m_Artboards.size())
```

Comment:
size_t should be unsigned so you may be able to skip the < 0

Reply:
Oh, good point; let me change that.